tests/integration/orders-update.spec.js

```diff
@@ -1,17 +1,93 @@
+import getApiCredentials from 'get-api-credentials'
 import OrdersUpdate from 'index'
-import { SphereClient } from 'sphere-node-sdk'
 import test from 'tape'
 
-let PROJECT_KEY
+import productTypeSample from '../helpers/product-type-sample.json'
+import productSample from '../helpers/product-sample.json'
+import orderSample from '../helpers/order-sample.json'
+import stateSamples from '../helpers/state-samples.json'
 
-if (process.env.CI === 'true')
-  PROJECT_KEY = process.env.CM_PROJECT_KEY
-else
-  PROJECT_KEY = process.env.npm_config_projectkey
+const PROJECT_KEY =
+  process.env.CM_PROJECT_KEY || process.env.npm_config_projectkey
 
-const logger = {
-  trace: () => {},
-  debug: () => {},
-  info: () => {},
-  error: () => {},
-}
+// Delete all API items from a given endpoint
+const deleteAllApiItems = (client, service) => client[service]
+  .all().fetch()
+  .then(data => data.body.results.filter(item => !item.builtIn))
+  .then(items => Promise.all(
+    items.map(item => client[service]
+      .byId(item.id)
+      .delete(item.version))
+  )
+)
+
+test('the module should modify an existing order', (t) => {
+  let ordersUpdate
+
+  getApiCredentials(PROJECT_KEY)
+    .then((apiCredentials) => {
+      ordersUpdate = new OrdersUpdate({
+        config: apiCredentials,
+      })
+    })
+    // Clean up, remove everything from used services in testing
+    .then(() => deleteAllApiItems(ordersUpdate.client, 'orders'))
```

Comment:
Good if we want to have another test suite in this file.

Best approach would have been to have a `before`/`after` API